tests/test_examples.py

```diff
@@ -7,6 +7,7 @@
 from typing import Iterable, Sequence
 
 import pytest
+from pytest_notebook import execution, notebook
```

Comment:
Can we change this to `import pytest_notebook` in line with our style guide: https://google.github.io/styleguide/pyguide.html#22-imports

I know this is an existing issue but may as well fix it